.github/ISSUE_TEMPLATE/5_performance_speed.md

```diff
@@ -46,6 +45,8 @@ assignees: ''
      so we know which functions might be causing it. See "How to Collect
      and Read Timeline Traces" on this blog post:
        https://medium.com/flutter/profiling-flutter-applications-using-the-timeline-a1a434964af3#a499
+     Make sure the performance overlay is turned OFF when recording the
+     trace (Pressing ‘P’ on the command line toggles the overlay).
```

Comment:
nit: We could mention that this is a useful tool for getting a general idea, but that the tool itself has more overhead than just tracing and tracing should be preferred.